pkgs/games/steam/steamcmd.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl, steam, bash
+, steamRoot ? "~/.local/share/Steam"
+}:
+
+stdenv.mkDerivation rec {
+  name = "steamcmd-${version}";
+  version = "20180104"; # According to steamcmd_linux.tar.gz mtime
+
+  src = fetchurl {
+    url = https://steamcdn-a.akamaihd.net/client/installer/steamcmd_linux.tar.gz;
```

Comment:
This is a little annoying it isn't versioned. Any way they could make one available? Definitely look into the url used by other distros.

Reply:
[AUR](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=steamcmd) does not have a versioned URL available.